frontend/app/tests/e2e/specs/accounts-balances/manual-balances.spec.ts

```diff
@@ -78,11 +76,12 @@ describe('balances', () => {
       });
       dashboardPage.getLocationBalances().then($dashboardBalances => {
         $dashboardBalances.forEach((dashboardBalances, index) => {
-          const balance = $manualBalances[index].renderedValue;
-          const dashboardBalance = dashboardBalances.renderedValue;
-          expect(dashboardBalance.toNumber()).within(
-            balance.minus(0.01).toNumber(),
-            balance.plus(0.01).toNumber()
+          const balance = $manualBalances[index];
+          const value = balance.value;
+          const dashboardBalance = dashboardBalances.value;
+          expect(dashboardBalance.toNumber(), balance.location).within(
+            value.minus(0.01).toNumber(),
+            value.plus(0.01).toNumber()
```

Comment:
Why not match it with previous file 
```
const { location, value } = $manualBalances[index];
const dashboardValue = dashboardBalances.value;
expect(dashboardValue.toNumber(), location).within(
    value.minus(0.01).toNumber(),
    value.plus(0.01).toNumber()
);
```

Sorry little bit OCD 🤣

Reply:
I only did it in the other place huh? good catch :)